Misc/NEWS.d/next/Library/2022-01-11-21-40-14.bpo-22833.WB-JWw.rst

```diff
@@ -0,0 +1,4 @@
+The inconsistent return types of :func:`email.header.decode_header` are now documented.
+
+:func:`email.header.decode_header_to_string` is provided as a less error-prone and
+more straightforward replacement for it.

```

Comment:
I suggest "alternative" instead of "replacement".

Reply:
Ack, will do.